### PR TITLE
feat(locale): add horses to pl

### PR DESCRIPTION
**Pull request by hankucz**

### CODE REVIEW

File: src/locales/pl/animal/horse.ts [hunk 1]
```diff
@@ -0,0 +1,242 @@
+export default [
```
Comment (pkuczynski) [inline src/locales/pl/animal/horse.ts:1]: It would be good to add a source, where this information comes from...

Comment (pkuczynski) [reply src/locales/pl/animal/horse.ts:1]: And the list should be sorted alphabetically...

**Review by pkuczynski — CHANGES_REQUESTED** [review]

**Comment by ST-DDT** [comment]
@hankucz Could you please perform the requested changes so we can merge this?

**Comment by Shinigami92** [comment]
@xDivisionByZerox did you progress on a PR for checking alphabetically sorted locales?
I think we should test it and add a manual filtering step to define if we want to list some specific cases where sorting doesn't make sense (like direction)

File: src/locales/pl/animal/horse.ts (hunk 1 above)
Comment (hankucz) [reply src/locales/pl/animal/horse.ts:2]: Done

**Comment by hankucz** [comment]
@ST-DDT feedback applied

**Review by pkuczynski — APPROVED** [review]

**Review by ST-DDT — APPROVED** [review]

**Review by Shinigami92 — APPROVED** [review]

**Pull request merged**